### DISCUSSION TITLE
Welcome to Solana PHP SDK Discussions!

### DISCUSSION THREAD

**Discussion by neverything** [verze-app/solana-php-sdk#36] [Announcements]
## 👋 Welcome frens!
  After a nice call with @mattstauffer who originally built this package, we are happy to announce, that we (https://verze.app, mostly @neverything) are going to maintain this package in the future.

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other users of this package.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this package.

## 🙇‍♂️ Thanks

  A big thank you to the original creator @mattstauffer and all the previous [contributors](https://github.com/verze-app/solana-php-sdk/graphs/contributors)

## 🔮 What's next?

- [x] Minor changes to the README
- [x] Making an announcement in [Discussions](https://github.com/verze-app/solana-php-sdk/discussions)
- [ ]  Housekeeping: Checking [current issues](https://github.com/verze-app/solana-php-sdk/issues) and [pull requests](https://github.com/verze-app/solana-php-sdk/pulls)
- [ ] Updating the [roadmap](https://github.com/verze-app/solana-php-sdk#roadmap)
- [ ] Changing the namespace to `Verzeapp`
- [ ] Making a new release and publishing it on

**Comment by noriods** [comment]
Thanks for this, guys!

I'm Norio and I'm a South African Laravel dev. I mostly do Laravel dev for US companies.

Right now I'm porting over a small Solana NFT Portfolio Tracker (in JS) to Laravel because I want to take advantage of Laravel's features like Socialite, Queues, etc.

The main thing I'll be using this package for, immediately, is getting all the transactions from a wallet.

In future, I may also be using this package for sending transactions but it's not an immediate need of mine.

Looking forward to seeing what you guys do with this. I'll try my best to help when and where I can but it'll probably look more like debugging and reporting issues than actual PRs.

Onwards and upwards, friends!
